src/grpc.ts

```diff
@@ -268,11 +268,17 @@ export class GrpcClient {
     const serviceAddress = options.servicePath + ':' + options.port;
     const creds = await this._getCredentials(options);
     const grpcOptions: ClientOptions = {};
+    // @grpc/grpc-js limits max receive message length starting from v0.8.0
```

Comment:
I would like to know more here.  Why is this something we need to override?  I trust it's the right thing to do, would just like to see more explanation and possibly links in the comment.

Reply:
I have updated the comment pointing to `@grpc/grpc-js` [release notes](https://github.com/grpc/grpc-node/releases/tag/%40grpc%2Fgrpc-js%400.8.0). This can be considered an allowed breaking change in `@grpc/grpc-js` since it was in 0.x range back then, but we cannot accept this breaking changes (it breaks Pub/Sub system tests, as an example).